<launch>

<arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
<arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />

  <!-- Experiment Playback bucket_video_test_2017-12-06-18-49-56.bag/ point_cloud_2018-05-04-15-42-45.bag-->
<param name ="/use_sim_time" value="true"/>  
<node pkg="rosbag" type="play" name="player" output="screen" 
args="--clock -r 0.02 -s 10 -u 3 /mnt/hgfs/Experiment_Data_MIT/_2018-11-12-16-27-18.bag"/>

<!-- fback_flow.cpp -->
<!-- <node name="fback_flow" pkg="opencv_apps" type="fback_flow" >
    <remap from="image" to="/camera1/image_raw" />
    <param name="use_camera_info" value="$(arg use_camera_info)" />
    <param name="debug_view" value="$(arg debug_view)" />
</node> -->

<!-- Mouse Click Test Node -->
<node pkg="opt_flow" type="BucketTipMappingCalibration.py" name="BucketTipMappingNode" output="screen">
        <remap from="UBC_Image"  to="/camera/color/image_raw" />
        <remap from="cal_Data"  to="/joint_states_dynamixel" /> <!-- /CalibratedJoints -->
        <remap from="PointCloud"  to="/camera/depth_registered/points" />
        <!-- <remap from="Crop_Flow"  to="/fback_flow/flows" /> -->
	<rosparam>
     </rosparam>
</node> 

</launch>